fzf-tab.zsh

```diff
@@ -97,8 +97,8 @@ _fzf_tab_remove_space() {
     --expect='/' # For continuous completion 
     '--color=hl:$(( $#headers == 0 ? 108 : 255 ))'
     --nth=2,3 --delimiter='\x00'  # Don't search FZF_TAB_PREFIX
-    --layout=reverse --height=70%
-    --tiebreak=begin -m --bind=tab:down,ctrl-j:accept,change:top,ctrl-space:toggle --cycle
+    --layout=reverse --height=75%
+    --tiebreak=begin -e -m --bind=tab:down,ctrl-j:accept,change:top,ctrl-space:toggle --cycle
```

Comment:
I'd like to keep a fuzzy match here. After all, it is fzf.
And please also edit README.md, in which I describe the default value of `FZF_TAB_OPTS`.